### PR TITLE
Various tweaks for visualization

### PR DESCRIPTION
**Pull request by zefciu**
* The racks get narrower on narrow monitors
* The hover/activation is uniformly done with a white border
* The borders in info table are corrected (removed extra border under
  last element.

### CODE REVIEW

File: src/ralph/ui/static/partials/data_center/rack.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<div class="rack rotate-{{ rack.orientation }} x-{{ rack.visualization_col }} y-{{ rack.visualization_row }}" ng-class="{'move': is_move, 'allowed': can_drop, 'not_allowed': !can_drop}">
+<div class="rack rotate-{{ rack.orientation }} x-{{ rack.visualization_col }} y-{{ rack.visualization_row }}" ng-class="{'active': rack.active, 'move': is_move, 'allowed': can_drop, 'not_allowed': !can_drop}">
     <a href="#/rack/{{ rack.id }}" class="wrapper" ng-show="mode == 'preview'">
-        <div class="name">
+        <div class="name">
```
Comment (kula1922) [inline src/ralph/ui/static/partials/data_center/rack.html]: why blank char?

File: src/ralph/ui/static/ui/js/app/data_center/controllers.js [hunk 2]
```diff
@@ -28,6 +28,10 @@ angular
         };
 
         $scope.$on('edit_rack', function (event, rack) {
+            angular.forEach($scope.data_center.rack_set, function(rack) {
```
Comment (kula1922) [inline src/ralph/ui/static/ui/js/app/data_center/controllers.js]: It is ok but for future you can use shortly way

$scope.data_center.rack_set.forEach(function(){...})

File: src/ralph/ui/static/ui/js/app/data_center/controllers.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ angular
...
+                rack.active = false;
+            });
+            rack.active = true;
```
Comment (kula1922) [inline src/ralph/ui/static/ui/js/app/data_center/controllers.js]: there is no way to active will be false. If previous loop set it like a false, here you back to true.

File: src/ralph/ui/static/partials/data_center/rack.html (hunk 1 above)
Comment (zefciu) [inline src/ralph/ui/static/partials/data_center/rack.html]: idk my bff jill. Will correct

**Comment by vi4m** [comment]
LGTM

**Comment by ar4s** [comment]
:ok:

**Pull request merged**